src/testRunner/unittests/sys/symlinkWatching.ts

```diff
@@ -126,23 +126,34 @@ describe("unittests:: sys:: symlinkWatching::", () => {
         return deferred.promise;
     }
 
+    function compareEventFileName(a: EventAndFileName["fileName"], b: EventAndFileName["fileName"]) {
+        if (a === null) a = undefined; // eslint-disable-line no-restricted-syntax
+        if (b === null) b = undefined; // eslint-disable-line no-restricted-syntax
+        return ts.compareStringsCaseSensitive(a, b);

```

Comment:
Ah, if that's all that's needed we can even remove the linter comments and just do this:
```suggestion
        return ts.compareStringsCaseSensitive(a ?? undefined, b ?? undefined);
```